gotocompany/guardian/v1beta1/guardian.proto

```diff
@@ -939,6 +939,7 @@ message Policy {
     string approve_if = 6;
     repeated string approvers = 7;
     string rejection_reason = 8;
+    bool dont_allow_self_approval = 9;
```

Comment:
would restrict_self_approval be a better name?

Reply:
IMO, dont_allow_self_approval is more appropriate. 
'restrict_self_approval' implies reducing the scope of self-approval but not fully eliminating it.
One more options is - 'disable_self_approval'

Reply:
how about `block_self_approval` or `deny_self_approval`?

Reply:
I understand `deny_self_approval` looks better in terms of technical language.
But we came up with `dont_allow_self_approval`  from the just opposite of `allow_self_approval`  as to maintain backward compatibility.
I think this name also gives the proper definition that self approval is not allowed and is fully eliminated. WDYT?